src/BubbleSort_array.md

```diff
@@ -35,3 +44,150 @@ function bubbleSort(A) {
   return A;
 }
 ```
+
+### Cpp
+
+```cpp
+void bubbleSort(int arr[]) {
+    int size = sizeof(arr) / sizeof(arr[0]);
+    for (int i = 0; i < size - 1; ++i) {
+        bool swapped = false;
+        for (int j = 0; j < size - i - 1; ++j) {
+            if (arr[j] > arr[j + 1]) {
+                int temp = arr[j];
+                arr[j] = arr[j + 1];
+                arr[j + 1] = temp;
+                swapped = true;
+            }
+        }
+        if (!swapped) {
+            break;
+        }
+    }
+}
+```
+
+### python
+
+```py
+def bubbleSort(arr):
+    n = len(arr)
+
+    for i in range(n):
+        for j in range(0, n-i-1):
+            if arr[j] > arr[j+1]:
+                arr[j], arr[j+1] = arr[j+1], arr[j]
+```
+
+### java
+
+```java
+public class BubbleSort {
+    public static void bubbleSort(int[] arr) {
```

Comment:
Just keep it a simple function, instead of class

Reply:
Done ♥